src/context.js

```diff
@@ -48,5 +48,14 @@ function getWorkspace() {
 
 }
 
+function getUserByApplicationKey(key){
```

Comment:
Could we call this parameter `applicationKey`? `key` is a bit vague. Per the google style docs, we should [use lowerCamelCase for parameters](https://google.github.io/styleguide/jsguide.html#naming-parameter-names).